en/mkdocs.yml

```diff
@@ -79,10 +77,6 @@ nav:
         - 'Overview': quick-starts/single-sign-on.md
         - 'OAuth/OIDC Apps': quick-starts/sso-for-oidc-apps.md
         - 'SAML Apps': quick-starts/sso-for-saml-apps.md
-      - 'Multi-Factor Authentication': 
-        - 'Possession Factor':  
-          - 'SMS OTP': quick-starts/mfa-sample.md
-          - 'X509': quickstarts/x509-sample.md
```

Comment:
shouldn't we link the x509.md file here?

Reply:
This is not tested with a sample

Reply:
let's keep it commented